runtimevar/runtimeconfigurator/runtimeconfigurator.go

```diff
@@ -157,8 +152,14 @@ func (w *watcher) WatchVariable(ctx context.Context) (driver.Variable, error) {
 func (w *watcher) ping(ctx context.Context) (*driver.Variable, error) {
 	// Use GetVariables RPC and check for deltas based on the response.
 	vpb, err := w.client.GetVariable(ctx, &pb.GetVariableRequest{Name: w.name})
-	w.lastRPCTime = time.Now()
 	if err != nil {
+		errCode := grpc.Code(err)
```

Comment:
Same here: please include in your commit message what was fixed here.

Reply:
Done.